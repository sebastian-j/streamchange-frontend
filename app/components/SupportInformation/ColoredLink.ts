import styled from 'styled-components';

const ColoredLink = styled.a`
  color: ${(props) => props.theme.color};
  display: block;
  font-family: 'Roboto', sans-serif;
  font-size: 14px;
  font-weight: 500;
  height: 20px;
  padding: 5px 0;
  text-decoration: none;
  text-transform: uppercase;
  span {
    padding: 0 8px 0 0;
  }
  svg {
    height: 24px;
    width: 24px;
  }
`;

export default ColoredLink;
