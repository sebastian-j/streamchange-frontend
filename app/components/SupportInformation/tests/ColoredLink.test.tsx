import React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import ColoredLink from '../ColoredLink';

const shallowRenderer = createRenderer();

describe('<ColoredLink />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<ColoredLink />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
