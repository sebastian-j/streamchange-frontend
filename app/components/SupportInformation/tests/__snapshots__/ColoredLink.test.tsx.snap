// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ColoredLink /> should render and match the snapshot 1`] = `
<a
  className="sc-bdvvtL cJBQlX"
/>
`;
